etc/PrimarySourceSetDataModelSample.json

```diff
@@ -43,7 +43,13 @@
     }
   ],
   "dct:temporal": [
-    { "name": "Colonial America" }
+    { 
+      "@id": "ushist:united-states-era-2",
+      "@type": "dct:PeriodOfTime",
```

Comment:
Probably we should use `edm:TimeSpan` (which is a subclass of `dct:PeriodOfTime`) for consistency.

Are `start` and `end` in some namespace? I'm missing the in Schema.org. Do we want `edm:begin/end`?